File: web/web/plugins/LinDevicePlugin.m
//
//  DeviceInfoPlugin.m
//  LinWeb
//
//  Created by lin on 4/10/15.
//  Copyright (c) 2015 lin. All rights reserved.
//

#import "LinDevicePlugin.h"

@implementation LinDevicePlugin
-(Json*)productName:(Json*)args{
    return [[Json alloc] initWithObject:@"iPhone"];
}
@end

File: web/web/plugins/LinDevicePlugin.swift
//
//  LinDevicePlugin.swift
//  LinWeb
//
//  Created by lin on 9/19/16.
//  Copyright © 2016 lin. All rights reserved.
//

import Foundation

open class LinDevicePlugin : LinAbstractWebPlugin{
//-(Json*)productName:(Json*)args{
//    return [[Json alloc] initWithObject:@"iPhone"];
    open func productName()->String{
        return "iPhone";
    }
//}
}
